.github/workflows/ci.yml

```diff
@@ -1,13 +1,28 @@
-name: Lint
-
 on: [push]
 
 jobs:
-  eslint:
+  js-lint:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
       - uses: stefanoeb/eslint-action@1.0.2
-
-      - name: js-unit-tests
+  js-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Build
+        run: yarn install
+      - name: Test
         run: yarn test
+  py-markdown-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Build
+        uses: docker://docker
+        with:
+          args: docker build ./test/markdown/ --tag=markdown-tests:latest
+      - name: Test
+        uses: docker://docker
+        with:
+          args: docker run markdown-tests:latest
```

Comment:
I tested this by editing the test to throw a `sys.exit(1)` early, and it appropriately failed this check